packages/shared/models/artifacts/StaticArtifacts.js

```diff
@@ -4,14 +4,12 @@ const BUILDS = {
   '@aragon/erc20-faucet': {
     'ERC20Faucet': require('@aragonone/erc20-faucet/build/contracts/ERC20Faucet'),
   },
-  '@aragon/minime': {
-    'MiniMeToken': require('@aragon/minime/build/contracts/MiniMeToken')
-  },
-  '@aragon/court': {
-    'AragonCourt': require('@aragon/court/build/contracts/AragonCourt'),
-    'DisputeManager': require('@aragon/court/build/contracts/DisputeManager'),
-    'JurorsRegistry': require('@aragon/court/build/contracts/JurorsRegistry'),
-    'CourtSubscriptions': require('@aragon/court/build/contracts/CourtSubscriptions'),
+  '@aragon/protocol-evm': {
```

Comment:
why are you calling this `aragon/protocol-evm`? this suggests that you are returning a dedicated virtual machine or something

Reply:
simply identifying the package for those artifacts